### ISSUE TITLE
Tensorflow 2.6 compatibility with previous versions?

### ISSUE THREAD

**Issue by ZiyueWangUoB** [tensorflow/tensorflow#53030]
Hi guys, I've trained a model using tensorflow 2.3, keras 2.3.1 and python 3.8. This model seems to be working perfectly fine in 2.4/2.5 but breaks with tf 2.6. I looked through the tensorflow 2.6 , as well as keras 2.6 release notes but couldn't find why this was breaking. 

The model can be loaded in tf.2.6 and the weights are all correct. However during inference, the result is different from that of tf2.3,2.4,2.5. Is there a reason for why this is? 

Model is a transfer learnt efficientnet B3, with a custom top made of dense layers and batch norm.

**Comment by Saduf2019** [comment]
@ZiyueWangUoB 
For keras related issue you may refer to this [link](https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999), Unless you fill the issue template and provide us with details/error tf version log we cannot help.
You may run your code on colab and share the gist.
